src/api/tests/auth/login/index.ts

```diff
@@ -12,11 +12,20 @@ export default (): void =>
       const user: User = body.user
       expect(user).toBeDefined()
       expect(user.uuid).toBeDefined()
+      expect(body.survey).not.toBeDefined() // only with param
       expect(user.email).toBe(mockUser.email)
       expect(user.authGroups?.length).toBe(1)
       expect(user.authGroups?.[0].name).toBe(AuthGroupName.systemAdmin)
     })
 
+    test('Login success with surveyId', async () => {
+      const { body } = await global.api
+        .post(`${ApiEndpoint.auth.login()}?includeSurvey=true`)
```

Comment:
pass includeSurvey to ApiEndpoint.auth.login() that gets appended to the url if it is defined